test/integration/targets/user/tasks/main.yml

```diff
@@ -36,6 +36,23 @@
 - import_tasks: test_expires_warn.yml
 - import_tasks: test_shadow_backup.yml
 - import_tasks: test_ssh_key_passphrase.yml
+- import_tasks: test_ssh_key_rounds.yml
+  when: (ansible_distribution == 'Fedora' and ansible_distribution_version is version('40', '>=')) or
+        (ansible_distribution == 'Ubuntu' and ansible_distribution_version is version('24.04', '>=')) or
+        (ansible_distribution == 'FreeBSD' and ansible_distribution_version is version('13.3', '>='))
+  vars:
+    passphrase: secret_passphrase
+    rounds: 100
```

Comment:
Would it be possible to lower this number, since the integration test will be slower and time-consuming? Ultimately we are just checking if the value of KDF works or not. Just a suggestion, not a blocker.

Reply:
This is a very good suggestion i think. I change it immediately. Thanks 😃